actors/builtin/multisig/multisig_actor.go

```diff
@@ -110,7 +110,14 @@ type ProposeParams struct {
 	Params []byte
 }
 
-func (a Actor) Propose(rt vmr.Runtime, params *ProposeParams) *cbg.CborInt {
+type ProposeReturn struct {
+	TxnID   cbg.CborInt
+	Applied bool
```

Comment:
When a transaction is proposed but not executed/applied we need a way to indicate that to the caller. One way the caller can be made aware of this is by using a bool here. This is also done for `ApproveReturn` since approving a transaction does not guarantee it is executed.